crp_APL/ctrl_vehicle_control/src/ctrl_vehicle_control.cpp

```diff
@@ -0,0 +1,151 @@
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <math.h>
+#include "rclcpp/rclcpp.hpp"
+#include "geometry_msgs/msg/pose.hpp"
+#include "std_msgs/msg/float32_multi_array.hpp"
+#include "std_msgs/msg/float32.hpp"
+#include "crp_msgs/msg/ego.hpp"
+
+#include "autoware_auto_control_msgs/msg/ackermann_control_command.hpp"
+#include "autoware_auto_planning_msgs/msg/trajectory.hpp"
+#include "autoware_auto_vehicle_msgs/msg/steering_report.hpp"
+#include "autoware_auto_vehicle_msgs/msg/vehicle_odometry.hpp"
+#include "nav_msgs/msg/odometry.hpp"
+#include "pacmod3_msgs/msg/vehicle_speed_rpt.hpp"
+#include "pacmod3_msgs/msg/system_rpt_float.hpp"
+
+#include "ctrl_vehicle_control/controller_inputs.hpp"
+#include "ctrl_vehicle_control/controller_outputs.hpp"
+#include "lib/utils.hpp"
+#include "compensatory_model/compensatory_model.hpp"
+
+using namespace std::chrono_literals;
+using std::placeholders::_1;
+
+class ctrl_vehicle_control : public rclcpp::Node
```

Comment:
```suggestion
class CtrlVehicleControl : public rclcpp::Node
```
Also definitions should be in a header file (.hpp)